log/log.go

```diff
@@ -55,6 +58,7 @@ func CleanExit(format string, v ...interface{}) {
 func Err(format string, v ...interface{}) {
 	fmt.Fprint(Stderr, pretty.Colorize("{{.Red}}[ERROR]{{.Default}} "))
 	fmt.Fprintf(Stderr, appendNewLine(format), v...)
+	ErrorState = true
```

Comment:
Practically from the stance of the code today? Nothing. It does satisfy the Go best practice of "Don't communicate by sharing memory, share memory by communicating", which might be pedantic. @technosophos mentioned this may be nice in the future too for bubbling up more specific errors/error codes later down the line, but I think that's beyond the scope we're talking about here.